Header header
float64 linear
float64 angular
uint8 mod_type
uint8 spin_turn